tema1/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html> ,<!DOCTYPE html>
```

Comment:
Pt un fisier html trebuie un singur element DOCTYPE

```suggestion
<!DOCTYPE html>
```